app/helpers/frontend/button_helper.rb

```diff
@@ -7,25 +7,48 @@ def goto_button(link: "#", icon: nil, add_class: "", target: "", title: nil, met
         title: title,
         data: tooltip_expando,
         method: method do
-          tag.span class: "#{icon} btn-icon__inner"
+          tag.span class: icon.to_s
         end
     end
 
-    def edit_button(link: "#", add_class: "")
+    def button_panel(id: nil, margin: "my-4", add_class: "", &block)
+      content_tag(:div, nil, id: id, class: "col-12 #{margin} d-flex justify-content-end #{add_class}", &block)
```

Comment:
Pretty sure that `content_tag :div` could be replaced with `tag.div`

Reply:
What is the benefit?

Reply:
Consistency with other areas of the codebase and terseness. Not much beyond that.